tests/test_agent_setup_prompt.py

```
@@ -0,0 +1,113 @@
+        'mkdocs': mkdocs,
+        'mkdocs.config': module_with_attributes('mkdocs.config', Config=object),
+        'mkdocs.structure': mkdocs_structure,
+        'mkdocs.structure.files': module_with_attributes('mkdocs.structure.files', Files=object),
+        'mkdocs.structure.pages': module_with_attributes('mkdocs.structure.pages', Page=object),
+    }
+    spec = importlib.util.spec_from_file_location('test_docs_plugins_main', REPO_ROOT / 'docs' / 'plugins' / 'main.py')
+    assert spec is not None and spec.loader is not None
+    module = importlib.util.module_from_spec(spec)
+    with patch.dict(sys.modules, mkdocs_modules):
+        spec.loader.exec_module(module)
+    return cast('Callable[[str, object], str]', getattr(module, 'agent_setup_prompt'))
```

Ruff (0.16.1)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)